Xwt.Gtk/Xwt.GtkBackend/CustomTreeModel.cs

```diff
@@ -178,8 +175,6 @@ public bool IterNext (ref Gtk.TreeIter iter)
 			TreePosition pos = NodeFromIter (iter);
 			TreePosition parent = source.GetParent (pos);
 			int i = GetIndex (parent, pos);
-			if (source.GetChildrenCount (parent) >= i)
-				return false;
```

Comment:
Why remove this validation?

Reply:
It's just overhead, see the lines after that, which just call the data source's .GetChild for the next sibling, which would return null if there is no child at that position, so there's no need to call this.

Also, AFAICS, the condition is wrong, as it checks if there are more children than the current position,while it should check if it has *less* to return `false` (i.e. no next iter).